src/truefoundry_sdk/agent_skills/raw_client.py

```
@@ -0,0 +1,572 @@
+                )
+                _items = _parsed_response.data
+                _has_next = True
+                _get_next = lambda: self.list(
+                    fqn=fqn,
+                    ml_repo_id=ml_repo_id,
+                    name=name,
+                    offset=offset + len(_items or []),
+                    limit=limit,
+                    include_empty_agent_skills=include_empty_agent_skills,
+                    request_options=request_options,
+                )
```

Ruff (0.15.9)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `_get_next` as a `def`

(E731)